pkgs/development/python-modules/python-fluent/fluent-runtime.nix

```
@@ -0,0 +1,39 @@
+{ aiohttp, attrs, babel, buildPythonPackage, fetchPypi, fluent-syntax, lib, pythonOlder, pytz }:
+
+buildPythonPackage rec {
+  pname = "fluent.runtime";
+  version = "0.4.0";
+  format = "wheel";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi rec {
+    inherit pname version format;
+    sha256 = "sha256-Uf0CWCwjY+EQbXBRZClnobf0Bt2cMXvUYApz7eQMUUY=";
+    dist = python;
+    python = "py2.py3";
+  };
+
+  propagatedBuildInputs = [
+    fluent-syntax
+    aiohttp
+    attrs
+    babel
+    pytz
+  ];
+
+  pythonImportsCheck = [
+    "fluent.syntax"
+    "attrs"
+    "babel"
+    "pytz"
+  ];
```

The import check should test if the import of the module works.

```suggestion
  pythonImportsCheck = [
    "fluent.runtime"
  ];
```